pom.xml

```diff
@@ -130,6 +130,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- Docker Agents -->
+    <dependency>
+      <groupId>org.jenkins-ci.test</groupId>
+      <artifactId>docker-fixtures</artifactId>
```

Comment:
This is deprecated, testcontainers is generally used instead

Reply:
Do we have an example?

Reply:
Lots of general examples:
https://github.com/search?q=org%3Ajenkinsci+testcontainers&type=code

or
https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Ajenkinsci+testcontainers

Reply:
Ah, it's https://www.testcontainers.org. I thought a new Jenkins approach. Thanks for the pointers!